.travis.yml

```diff
@@ -39,8 +39,21 @@ jobs:
         - env
         - go test -coverprofile coverage.out $(go list ./... | grep -v tests)
         - go tool cover -func=coverage.out
+    - stage: bdd tests
+      services:
+        - docker
+      before_script:
+        - go build
+        - wget -O docker-compose.yml https://raw.githubusercontent.com/RedHatInsights/insights-behavioral-spec/main/docker-compose.yml
+        - POSTGRES_DB_NAME=notification docker-compose --profile test-notification-services up -d
+        - cid=$(docker ps | grep 'insights-behavioral-spec:ci' | cut -d ' ' -f 1)
+        - docker cp ccx-notification-service $cid:`docker exec $cid bash -c 'echo "$VIRTUAL_ENV_BIN"'`
+        - docker exec -u root $cid /bin/bash -c 'chmod +x $VIRTUAL_ENV_BIN/ccx-notification-service'
+      script:
```

Comment:
i will approve but we could consider creating a `travis-bdd-test.sh` file to encapsulate this?

Reply:
makes sense. I'll refactor in a follow-up PR to not mess this one.